src/managers/GuildApplicationCommandManager.js

```diff
@@ -119,6 +120,87 @@ class GuildApplicationCommandManager extends ApplicationCommandManager {
     );
   }
 
+  /**
+   * Add permissions to a command.
+   * @param {ApplicationCommandResolvable} command The command to edit the permissions for
+   * @param {ApplicationCommandPermissionData[]} permissions The permissions to add to this command
+   * @returns {Promise<ApplicationCommandPermissions[]>}
+   * @example
+   * // Block the @everyone role from command permissions
+   * guild.commands.addPermissions('123456789012345678', [
+   *   {
+   *     id: guild.roles.everyone.id,
+   *     type: 'ROLE',
+   *     permission: false
+   *   },
+   * ])
+   *   .then(console.log)
+   *   .catch(console.error);
+   */
+  async addPermissions(command, permissions) {
+    const id = this.resolveID(command);
+    if (!id) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+
+    const perms = await this.fetchPermissions(id).catch(() => []);
```

Comment:
I know this is to handle the 404 when no permissions are set, but you should probably check the error code or something first - that's not the *only* reason a Promise could reject.